src/components/lessons/Git.md

```diff
@@ -45,6 +43,112 @@ git log
 
 ## Lets get started
 
-So now we have git installed it is time to start using it. First we navigate to www.github.com and create a profile. If you already have one that is great. We can then go ahead and create a new repository. You can name this anything you like. It is good to have a README file in your repository 
+So now we have git installed it is time to start using it. We have a few ways we can use GitHub on our machine. We can use GitHub desktop or we can use it in the terminal. We will be using the terminal for this tutorial. First we navigate to www.github.com and create a profile.  We can then go ahead and create a new repository. You can name this anything you like. It is good to have a README file in your repository as it can give information on what technologies you are using and what you are trying to do with you project.
 
+Now you have created a new repo (short for repository) we will clone it on to our local machine. We first need the address for our repository which we can get by clicking on the new repo and on the green Code button. You should be able to copy the address from there. Once we have that we can do in to our text editor and open up our terminal. 
+
+## git clone
+
+To clone our repo we will type git clone (paste the address or your repo hero) and then press enter e.g :
+
+git clone git@github.com:Your-Name-Here/your-project.git
+
+This will create a copy of your repo . From here once we are in the folder You can go ahead and create and edit all you like. Once we are ready to save some changes we move on to git add.
+
+## git add
+
+If we type:
+
+“Git add .”
+
+It will tell git to track all of the files that are listed in the project. Alternatively we could just add the name of the file or folder we want it to track e.g:
```

Comment:
If you're using it in command like, write **git**. If you're referring to Git as a name of the software, capitalize it instead. So the sentence should be like "It will tell Git to track..."